website_seo_sale/models/product.py

```diff
@@ -0,0 +1,41 @@
+from openerp.osv import osv, fields
+from openerp.addons.website_seo_sale.models.website import slug
+
```

Comment:
If is possible you should try to use apiv8. Use Model from openerp.models and remove arguments 'cr', 'uid', ... using api decorators.